### ISSUE TITLE
Newest version of PyGithub is not compatible with the bot

### ISSUE THREAD

**Issue by laraPPr** [EESSI/eessi-bot-software-layer#223]
PyGithub recently realease PyGithub 2.1.1 which is no longer compatible with the bot. The last version that of PyGithub that is compatible with the bot is v1.59.1. [link](https://pypi.org/project/PyGithub/#history)

With PyGithub 2.1.1 the bot has the following error when trying to submit more than one comment to a pr (It can respond to one bot command but when a seccond command is submitted in the pr the bot cannot respond to it).

The error: 

```
Traceback (most recent call last):
  File "/kyukon/scratch/gent/461/vsc46128/EESSI/venv_eessi_bot_login_p38/lib/python3.8/site-packages/pyghee/lib.py", line 170, in process_event
    self.handle_event(event_info, log_file=log_file)
  File "/kyukon/scratch/gent/461/vsc46128/EESSI/venv_eessi_bot_login_p38/lib/python3.8/site-packages/pyghee/lib.py", line 102, in handle_event
    handler(event_info, log_file=log_file)
  File "/kyukon/scratch/gent/461/vsc46128/EESSI/eessi-bot-software-layer/eessi_bot_event_handler.py", line 208, in handle_issue_comment_event
    issue_comment = create_comment(repo_name, pr_number, comment_body)
  File "/kyukon/scratch/gent/461/vsc46128/EESSI/eessi-bot-software-layer/tools/pr_comments.py", line 45, in create_comment
    gh = github.get_instance()
  File "/kyukon/scratch/gent/461/vsc46128/EESSI/eessi-bot-software-layer/connections/github.py", line 104, in get_instance
    if not _gh or (_token and datetime.datetime.utcnow() > _token.expires_at):
TypeError: can't compare offset-naive and offset-aware datetimes
```

We should try to resolve this error so that the bot uses the latest version of PyGitlab. 

If we implement the fix we should also update `requirements.txt` so that we at least require PyGithub version 2.1.1.

**Comment by laraPPr** [comment]
Fixed by changing the datetime. But requires PyGithub to be at least 2.1.1

```
from datetime import datetime, timezone
if not _gh or (_token and datetime.now(timezone.utc) > _token.expires_at):
```

**Comment by boegel** [comment]
Seems like the problems we're seeing are caused by a documented breaking change in PyGithub v2.1.0, see https://github.com/PyGithub/PyGithub/releases/tag/v2.1.0.post0

It seems to boil down to https://github.com/PyGithub/PyGithub/pull/2565

**Comment by trz42** [comment]
Fixed with #224, included in https://github.com/EESSI/eessi-bot-software-layer/releases/tag/v0.1.1
